roles/ceph-rgw/tasks/pre_requisite.yml

```diff
@@ -41,6 +41,12 @@
     group: "{{ key_group }}"
   when: cephx
 
+- name: trigger creation of ceph-radosgw systemd unit file
+  command: chkconfig --add ceph-radosgw
```

Comment:
Yes, I believe it is, although I'm not an expert in this area. If you prefer, it could be made conditional on the ceph-radosgw.service file not being present.